lib/Mojo/Loader.pm

```diff
@@ -150,6 +170,14 @@ Check if embedded file from the C<DATA> section of a class was Base64 encoded.
 
 Search for packages in a namespace non-recursively.
 
+=head2 is_package
+
+  if( is_package('a1::aaa') ){
+    say "package name valid";
+  }
+
+Check package name.
```

Comment:
Inconsistent documentation.

Reply:
Sorry, my english isn't good for write pod. Could you please write text here? I'll insert it in request.

Reply:
Not just the text, also the code.

Reply:
Simplistic text i don't mind too much, as long as it's phrased correctly. But the rest needs to be consistent with the rest of the docs.

Reply:
This would be more consistent.

```pod
=head2 is_package

  my $bool = is_package "Mojo::Base";
  # "Incorrect name for package"
  say "Incorrect name for package" unless is_package "MyApp.pm";

Check if value is a package name.
```

Reply:
No it wouldn't at all @kiwiroy. We never have comments in the calling convention section.

Reply:
I don't think much tweaking is needed here. How about something like this?
```
=head2 is_package

  say "Package name is valid" if is_package "Mojo::Base";

Check if a provided string is a valid package name. Please note that this only verifies the name of the package. It does not check if the package exists or is loaded.
```

I changed the example to a test returning something truthy instead of something falsey. Also the description text I provided I am still unsure of, specifically the latter sentence. Any input from you @kraih is appreciated.
```

Reply:
The convention here is:
```perl
=head2 is_package

  my $bool = is_package 'Mojo::Base';

Check if value is a package name.

  # False
  is_package 'MyApp.pm';
  is_package 'My/App.pm';

  # True
  is_package 'MyApp';
  is_package 'My::App';
```

Reply:
Yeah, that is _much_ better :+1: 

@greenfactory Could you update the code accordingly?